### ISSUE TITLE
Stats collection during initial launch resulted in error 502 and hung up the startup process

### ISSUE THREAD

**Issue by addianto** [Kareadita/Kavita#1214]
**Describe the bug**

Hello!

I encountered a problem when trying to run Kavita for the first time.
The startup process got stuck after the stats collection task failed to call the API at https://stats.kavitareader.com/api/v2/stats.
The API call failed and returned exception with status code 502 (Bad Gateway).
The startup process did not continue and perpetually stuck until I terminated Kavita manually using CTRL-C.

**To Reproduce**

1. Download the latest release (0.5.2.5) and extract the TAR archive into a directory
2. `cd` to directory where the TAR was extracted and run `./Kavita` command
3. Let the startup process executes the database migration and other tasks and wait until it executed stats collection task
4. See the error messages about the `FlurlHttpException` that raised during the request to `KavitaStats`

**Expected behavior**

The stats collection task successfully sent the data to the API and the startup process proceed normally.

OR

If the stats collection task failed due to API unavailability, at least let the startup process continues.

**Screenshots**

N/A.

**Environments**

I tested Kavita on two environments: Raspberry Pi 3 Model B+ running Arch Linux ARM and Ubuntu VM on Google Cloud Platform.
I tried Kavita containers and binary releases on both environments.

**Comment by majora2007** [comment]
The start server went down earlier but it's interesting your server hung. When I tested, I didn't hang at all, of course the log showed that last, but the UI was still useable. 

What's also interesting is that stat reporting is scheduled after the startup bootstraps, so startup should always occur. 

I'll double check that there is no way for the service to hang in case stats goes down.

**Comment by addianto** [comment]
I just tried re-deploying Kavita again both in my Raspberry Pi and VM on Google Cloud, and the startup process proceed normally. 😅 Maybe the API (https://stats.kavitareader.com/api/v2/stats) was down or in maintenance when I tried deploying Kavita few days ago.

Is it possible to make the stats collection configurable from environment variable or static file configuration? It might be beneficial for debugging and also for users who prefer to opt out the stats collection.
